src/main/resources/data/retail/pockets.json

```diff
@@ -14,7 +14,7 @@
     "icon": "home",
     "goal": {
       "amountCurrency": {
-        "amount": "800.00",
```

Comment:
The user became poor suddenly :stuck_out_tongue:

Reply:
There were 2 goals with the same amount, so I change one to have more difference.